### DISCUSSION TITLE
memos如何 保存前一次部署的数据

### DISCUSSION THREAD

**Discussion by JesseJeson** [zeabur/.github#11] [Q&A] [answered]
采用的是fork memos官方仓库的部署方式

**Comment by yuaanlin** [comment, marked as answer]
您好，fork 倉庫的部署方式將無法保存數據，因為 Memos 是將數據保存在本地文件中，而 Zeabur 要求從 git 部署的服務必須是「無狀態」的，因此當您將數據存放到本地文件後，下次部署這些數據就會被清空。

建議您使用 Zeabur 的 Service Marketplace 服務市場中的 Memos 來部署，這樣就可以解決數據無法持久的問題！

**Reply by JesseJeson** [reply]
好的谢谢
